gdal/configure

```diff
@@ -41358,6 +41363,27 @@ RASDAMAN_LIB=$RASDAMAN_LIB
 
 
 
+
+# Check whether --with-brunsli was given.
+if test "${with_brunsli+set}" = set; then :
+  withval=$with_brunsli;
+fi
+
+
+BRUNSLI_ENABLED=no
+
+if test "$with_brunsli" != "" -a "$with_brunsli" != "no" ; then
+  BRUNSLI_ENABLED="yes"
+  BRUNSLI_ENABLED=$BRUNSLI_ENABLED
+
+  BRUNSLI_INCLUDE=-I$prefix/include
+
+  BRUNSLI_LIB="-lbrunslicommon -lbrunslienc -lbrunslidec -lbrotlicommon -lbrotlienc -lbrotlidec"
```

Comment:
`BRUNSLI_LIB` is super fragile:

- lib ordering is wrong brotlicommon is a dependency of brotlienc and brotlidec, same for brunsli

- Lib names of brunsly seem wrong: https://github.com/google/brunsli/blob/v0.1/brunsli.cmake#L50-L85

- brolti lib names are different if static (`-static` suffix)